rb/spec/unit/selenium/webdriver/remote/http/curb_spec.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+# Licensed to the Software Freedom Conservancy (SFC) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The SFC licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+require File.expand_path('../../spec_helper', __dir__)
+require 'selenium/webdriver/remote/http/curb'
+begin
+  require 'curb'
+rescue LoadError
+  raise 'Curb is not supported by this platform. Try installing Curl'
```

Comment:
It's just a spec, we don't really need that.

Reply:
Okey, I will roll back to the comment now